plugins/lime-plugin-notes/src/notesPage.js

```diff
@@ -36,9 +36,9 @@ export const Page = ({ setNotes, getNotes, notes, hostname, loading }) => {
 	}, [notes]);
 
 	return (
-		<div className="container" style={{ paddingTop: '100px' }}>
+		<div class="container" style={{ paddingTop: '100px' }}>
 			<h4><span>{I18n.t('Notes of')}</span> {hostname}</h4>
-			<textarea onChange={handleChange} className={style.notes} value={value} />
+			<textarea onChange={handleChange} class={'notes'} value={value} />
```

Comment:
We should keep className instead of class since class is a reserved word in js, shouldn't we?

Reply:
There seems to be a problem with importing styles and if I use className it does not apply the style to the textarea.